tools/macOS/builder

```diff
@@ -222,6 +222,7 @@ table_line_append DIST_PRE_BUILD ffmpeg "sed -i.bak 's/-lstdc++/-lc++/g' configu
 # For a 10.7 target:
 #table_line_replace DISTS wxwidgets "https://github.com/wxWidgets/wxWidgets/releases/download/v3.0.5.1/wxWidgets-3.0.5.1.tar.bz2 lib/libwx_baseu-3.0*.a"
 
+table_line_append DIST_ARGS sdl3  "-DCMAKE_C_FLAGS=-fno-fast-math"
```

Comment:
This would be better in `DIST_EXTRA_CFLAGS` near `DIST_EXTRA_CXXFLAGS` in `core.sh`.

Reply:
I'll apply that after my nap